adapters/blip_2/blip_2.py

```diff
@@ -0,0 +1,79 @@
+import json
+from typing import List
+import PIL
+import dtlpy as dl
+import logging
+from transformers import AutoProcessor, Blip2ForConditionalGeneration
+
+logger = logging.getLogger("[BLIP]")
+CAPTIONING_PROMPT = "Caption this image."
+
+
+class HuggingAdapter:
+    def __init__(self, configuration):
+        self.model_name = configuration.get("model_name")
+        self.device = configuration.get("device")
+        self.conditioning = configuration.get("conditioning", False)
+        self.processor = AutoProcessor.from_pretrained("Salesforce/blip2-opt-2.7b")
+        self.model = Blip2ForConditionalGeneration.from_pretrained("Salesforce/blip2-opt-2.7b")
+        self.model.to(self.device)
+
+    def prepare_item_func(self, item: dl.Item):
```

Comment:
this is the main function that was simplified with the addition of prompt items in the SDK

Reply:
Done ✅ 

The adapter is refactored to use prompt items